ui/src/components/CardInput.js

```diff
@@ -40,7 +40,7 @@ class CardInput extends Component {
             url: '/v1/api/game/' + this.props.gameId + '/card',
             timeout: 4000,    // 4 seconds timeout
             data: {
-                value: this.state.card,
+                value: capitalize(this.state.card),
```

Comment:
I thought we should just capitalize the card values on input (once) instead of render (four times)